dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java

```diff
@@ -117,9 +117,12 @@ private boolean isDBHealthy(final long timeOut) throws Throwable {
                 .get(this.failFastBooleanPolicy(timeOut, () -> {
                     try{
                         final DotConnect dc = new DotConnect();
-                        dc.setSQL("select count(*) as count from contentlet");
-                        final List<Map<String,String>> results = dc.loadResults();
-                        return Long.parseLong(results.get(0).get("count")) > 0;
+                        if(DbConnectionFactory.isPostgres()) {
+                            return dc.setSQL("SELECT count(*) as count FROM (SELECT 1 FROM dot_cluster LIMIT 1) AS t").getInt("count")>0;
```

Comment:
why is not it  just do 'SELECT 1 FROM dot_cluster LIMIT 1'?, really we don't need to count, also we can do it for every data base.

Reply:
It is as lightweight as we can make it while trying to avoid a full table scan in postgres.  The switch to using the `dot_cluster` table means that at most there will be 1 record in any dotCMS db which should be good enough for most other dbs.

Reply:
do you think this improvement applies for others DBs as well?

Reply:
"The switch to using the dot_cluster table means that at most there will be 1 record in any dotCMS db", that is why I said that really we don't need to count

Reply:
A count is going to obviously be performant in a table with 1 record.  I think it is ok as is.